check-code-style/action.yml

```diff
@@ -64,7 +64,20 @@ runs:
         ${{ github.action_path }}/check_style_bzl.sh
       shell: bash
 
+    - name: Determine whether there are any Python files
```

Comment:
```suggestion
    # TODO(rjh): consider replacing this with 'dorny/paths-filter@v2' rather than us rolling our own
    # filter checking code.
    - name: Determine whether there are any Python files
```